src/modules/exec.ts

```diff
@@ -76,7 +85,8 @@ export async function runExecSpec(
     componentId: string,
     projectConfig: ProjectConfig,
     envVars: NodeJS.ProcessEnv,
-    verbose?: boolean
+    writeStdout: boolean,
```

Comment:
what do you think of having an object here? For better understanding when setting this?
sth like stdOutOptions: {verboseLog:boolean, writeStdOut:boolean}